_internals/configuration.md

```diff
@@ -118,329 +118,329 @@ When `ANSI_DEFAULTS` is enabled, it enables the following ISO settings:
 
-`OFF` setting is not allowed for this option. Please use 
+Setting to `OFF` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.arithignore
+### `babelfishpg_tsql.arithignore`
 
-`ON` setting is not allowed for this option. Please use 
+The setting `ON` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.concat_null_yields_null
+### `babelfishpg_tsql.concat_null_yields_null`
 
 If enabled, concatenating a `NULL` value produces a `NULL` result.
 
 
-### babelfishpg_tsql.cursor_close_on_commit
+### `babelfishpg_tsql.cursor_close_on_commit`
 
-`ON` setting is not allowed for this option. Please use 
+The setting `ON` is not allowed for this option. Please use
 `babelfishpg_tsql.escape_hatch_session_settings` to ignore errors.
 
 
-### babelfishpg_tsql.database_name
+### `babelfishpg_tsql.database_name`
 
 Predefined Babelfish database name. The default value is `babelfish_db`.
 
 
-### babelfishpg_tsql.datefirst
+### `babelfishpg_tsql.datefirst`
 
 Sets the first day of the week to a number from 1 through 7.
 
 
-### babelfishpg_tsql.default_locale
+### `babelfishpg_tsql.default_locale`
 
 The default locale to use when creating a new collation. The default value is
-`en_US`. Only superusers can set this variable. 
+`en_US`. Only superusers can set this variable.
 
 
-### babelfishpg_tsql.disable_batch_auto_commit
+### `babelfishpg_tsql.disable_batch_auto_commit`
 
 Disable auto commit inside procedures. The default value is `false`.
 
 
-### babelfishpg_tsql.disable_internal_savepoint
+### `babelfishpg_tsql.disable_internal_savepoint`
 
 Disable internal savepoints. The default value is `false`.
 What is the purpose of this variable? It controls whether a savepoint is created to
 handle undo if a command fails. Savepoints are not started for batch commands, as
 error handling must be taken care of at the statement level.
 
 
-### babelfishpg_tsql.disable_txn_in_triggers
+### `babelfishpg_tsql.disable_txn_in_triggers`
 
-Disable transactions in triggers. The default value is `false`. 
+Disable transactions in triggers. The default value is `false`.
 TSQL triggers terminate if there is no transaction active at the end.
 
 
-### babelfishpg_tsql.dump_antlr_query_graph
+### `babelfishpg_tsql.dump_antlr_query_graph`
 
 Dump query graph parsed by ANTLR parser to local disk. The path used is
 `/tmp/antlr.dot`.
 
 
-### babelfishpg_tsql.enable_antlr_detailed_log
+### `babelfishpg_tsql.enable_antlr_detailed_log`
 
 Enable detailed ATNLR parser logging.
 
 
-### babelfishpg_tsql.enable_ownership_structure
+### `babelfishpg_tsql.enable_ownership_structure`
 
-Enable Babelfish ownership structure. 
+Enable Babelfish ownership structure.
 
 
-### babelfishpg_tsql.escape_hatch_compatibility_level
+### `babelfishpg_tsql.escape_hatch_compatibility_level`
 
 Defines compatibility level.
```

Comment:
It does not. this escape hatches suppresses the error when trying to set the compatibility level.

Reply:
Changed accordingly.